haskell/src/Lorentz/Contracts/Stablecoin/Types.hs

```diff
@@ -127,33 +127,47 @@ deriving anyclass instance HasAnnotation SetExpiryParam
 instance Buildable SetExpiryParam where
   build = genericF
 
-instance HasAnnotation FA2.Parameter where
+-- | Similar to 'FA2.Parameter' from @morley-ledgers@, but with a layout compatible with
+-- the ligo implementation
+data FA2Parameter
```

Comment:
Oh I thought (referring to the whole commit) that this would have been unnecessary after [this recent MR](https://gitlab.com/morley-framework/morley-ledgers/-/merge_requests/30).

Is this only because of the HLS you mentioned then? Like, would it compile fine?

Reply:
Yeah, it compiles fine, it's just that it breaks HLS :/

Reply:
Ok I guess this is fine anyway, both solutions are kinda meh, if possible I guess we should look into how to avoid the problem in the first place :thinking:

Reply:
I started a discussion [here](https://serokell.slack.com/archives/CF2UVU8RX/p1610462545046100), I'd appreciate your input :)